scripts/restoreDump.js

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env babel-node
+
+import fs from 'fs'
+import { execSync } from 'child_process'
+import { Log, env, cli } from 'decentraland-commons'
+
+import { loadEnv, resolvePath, runpsql } from './utils'
+
+const log = new Log('restoreDump')
+
+export async function restoreDump(dumpPath) {
+  if (!dumpPath) {
+    throw new Error(
+      'Empty dump path. Please add it as an argument when running this script'
+    )
+  }
+
+  const connectionString = env.get('CONNECTION_STRING')
+  if (!connectionString) {
+    throw new Error('Connection string empty. Please fill your env file')
+  }
+
+  const shouldContinue = await cli.confirm(
+    `Careful! this will DROP your database and re CREATE it
+Do you wish to continue?`
+  )
+  if (!shouldContinue) return process.exit()
+
+  const databaseName = connectionString.split('/').pop()
+  const dumpFullPath = resolvePath(dumpPath)
+
+  if (!fs.existsSync(dumpFullPath)) {
+    throw new Error(`Couldn't find dump file "${dumpFullPath}"`)
+  }
+
+  log.info(`Reseting ${databaseName}`)
+  execSync(`psql -c 'DROP DATABASE IF EXISTS ${databaseName}'`)
+  execSync(`psql -c 'CREATE DATABASE ${databaseName};'`)
+
+  log.info(`Running dump ${dumpFullPath}`)
+  execSync(runpsql(dumpPath))
+
+  log.info('Migrating')
+  execSync('npm run migrate up')
+
+  log.info('All done!')
+  process.exit()
+}
+
+if (require.main === module) {
+  loadEnv()
+
+  const dumpPath = process.argv.length > 2 ? process.argv.pop() : ''
+
+  Promise.resolve()
+    .then(() => restoreDump(dumpPath))
+    .catch(console.error)
```

Comment:
I am wondering if we can add a `process.exit()` in the `catch` clause.

I guess the process will continue open with the error message displayed and we will need to stop it manually (cmd + c).

Reply:
Nope, the process ends, that catch is there to avoid a `Uncatched Promise error` and to log out the error:

![image](https://user-images.githubusercontent.com/692077/55320538-96d69f00-5477-11e9-8676-f07f998adb05.png)